symbology/read-symbols-from-mobile-style-file/src/main/resources/read_symbols_from_mobile_style_file.fxml

```diff
@@ -0,0 +1,80 @@
+  ~ distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  ~ WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  ~ License for the specific language governing permissions and limitations under
+  ~ the License.
+  -->
+
+<?import javafx.geometry.Insets?>
+<?import javafx.scene.control.ListView?>
+<?import javafx.scene.control.Slider?>
+<?import javafx.scene.layout.Background?>
+<?import javafx.scene.layout.BackgroundFill?>
+<?import javafx.scene.layout.HBox?>
+<?import javafx.scene.layout.StackPane?>
+<?import javafx.scene.layout.VBox?>
+<?import javafx.scene.paint.Color?>
+<?import com.esri.arcgisruntime.mapping.view.MapView?>
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.control.Button?>
+<StackPane fx:controller="com.esri.samples.read_symbols_from_mobile_style_file.ReadSymbolsFromMobileStyleFileController"
+           xmlns:fx="http://javafx.com/fxml" stylesheets="/style.css">
+    <!--SDK MapView-->
+    <MapView fx:id="mapView" onMouseClicked="#handleMouseClicked"/>
+    <!--Control Panel-->
+    <VBox StackPane.alignment="TOP_LEFT" maxHeight="380" maxWidth="310" styleClass="panel-region" spacing="10">
+        <!--Control Panel Styling-->
+        <padding>
+            <Insets topRightBottomLeft="10"/>
+        </padding>
+        <background>
+            <Background>
+                <fills>
+                    <BackgroundFill>
+                        <fill>
+                            <Color opacity="0.5"/>
+                        </fill>
+                    </BackgroundFill>
+                </fills>
+            </Background>
+        </background>
+        <HBox spacing="10" alignment="CENTER">
+            <VBox>
+                <Label>Hat:</Label>
+                <ListView fx:id="hatSelectionListView" maxHeight="120" maxWidth="75"/>
+            </VBox>
+            <VBox>
+                <Label>Eyes:</Label>
+                <ListView fx:id="eyesSelectionListView" maxHeight="120" maxWidth="60"/>
+            </VBox>
+            <VBox>
+                <Label>Mouth:</Label>
+                <ListView fx:id="mouthSelectionListView" maxHeight="120" maxWidth="60"/>
+            </VBox>
+            <VBox>
+                <Label>Color:</Label>
+                <ListView fx:id="colorSelectionListView" maxHeight="120" maxWidth="60"/>
+            </VBox>
+        </HBox>
+        <HBox>
+            <Label>Size:</Label>
+            <Slider fx:id="sizeSlider" min="10" value="50" majorTickUnit="1" minWidth="200" onMouseDragged="#updateSymbolSize" onMouseReleased="#updateSymbolSize"/>
+            <Label fx:id="sizeLabel" text="50 px"/>
+        </HBox>
+        <HBox alignment="CENTER_LEFT">
+            <Label>Preview:</Label>
+            <HBox alignment="CENTER" fx:id="symbolPreviewHBox" minHeight="140" minWidth="140"/>
+        </HBox>
+        <Button text="Reset" onAction="#resetView" minWidth="80"/>
```

Comment:
'Reset' makes me thing the symbol preview will be reset. I think 'Clear' would be more appropriate.